src/sql/parts/common/customInputConverter.ts

```diff
@@ -63,16 +63,8 @@ export function convertEditorInput(input: EditorInput, options: IQueryEditorOpti
 				let providerIds: string[] = [DEFAULT_NOTEBOOK_PROVIDER];
 				if (input) {
 					fileName = input.getName();
-					providerIds = getProvidersForFileName(fileName, notebookService);
```

Comment:
Do we have a plan to remove the package.json contribution letting you support non-IPYNB files here? Shouldn't we just pass in the providerId and providers to the NotebookInput now so it can set it appropriately in the NotebookEditorModel?